File: ComonLive/app/src/main/java/com/creator/comonlive/util/AlertUtil.java
package com.creator.comonlive.util;

import android.app.Activity;
import android.app.AlertDialog;
import android.content.DialogInterface;

import com.creator.comonlive.R;

/**
 * Created by skcrackers on 5/27/16.
 */
public class AlertUtil {
    public static void showConfirmDialog(final Activity activity, final String title, final String message) {
        AlertDialog.Builder alertbox = new AlertDialog.Builder(activity);
        alertbox.setTitle(title);
        alertbox.setCancelable(false);
        alertbox.setMessage(message);
        alertbox.setNeutralButton(activity.getResources().getString(R.string.confirm), new DialogInterface.OnClickListener() {
            public void onClick(DialogInterface dialog, int which) {
                activity.finish();
            }
        });

        alertbox.show();
    }
}

File: 2026/OneLive/app/src/main/java/com/creator/onelive/util/AlertUtil.kt
package com.creator.onelive.util

import android.app.Activity
import android.app.AlertDialog
import com.creator.onelive.R

/**
 * Created by skcrackers on 5/27/16.
 */
object AlertUtil {
    fun showConfirmDialog(activity: Activity, title: String?, message: String?) {
        val alertbox = AlertDialog.Builder(activity)
        alertbox.setTitle(title)
        alertbox.setCancelable(false)
        alertbox.setMessage(message)
        alertbox.setNeutralButton(activity.resources.getString(R.string.confirm)) { dialog, which -> activity.finish() }
        alertbox.show()
    }
}